Automattic/gridicons
# Build: Update/php templates in sources

In the refactoring #217 I missed to move the PHP source templates under `sources/`. This PR does that.

Also fixes a minor comment.

### To Test

1. Run `grunt`
2. Verify the PHP file gets generated correctly (compare with before this PR) in `php/gridicons.php`.
